lib/pf/Switch/Aruba/CX.pm

```diff
@@ -202,8 +202,6 @@ sub acl_chewer {
     my $i = 0;
     my $acl_chewed;
     foreach my $acl (@{$acl_ref->{'packetfence'}->{'entries'}}) {
-        #Bypass acl that contain tcp_flag, it doesnt apply correctly on the switch
-        next if (defined($acl->{'tcp_flags'}));
```

Comment:
revert please

Reply:
CX.pm is now ArubaOS_CX_10.x.pm I removed CX.pm